example/purchase-tester/src/components/FunctionTesterContainer.tsx

```diff
@@ -2,18 +2,19 @@ import './FunctionTesterContainer.css';
 import {IonButton, IonCard, IonCardContent, IonCardHeader, IonCardSubtitle, IonCardTitle} from "@ionic/react";
 import React, {useEffect, useState} from "react";
 import {
-  LOG_LEVEL,
-  PRODUCT_CATEGORY,
   PurchaseDiscountedPackageOptions,
   Purchases
 } from "@revenuecat/purchases-capacitor";
+
 import {REVENUECAT_API_KEY} from "../constants";
+import {LOG_LEVEL, PRODUCT_CATEGORY} from "@revenuecat/purchases-typescript-internal";
```

Comment:
Is this change required for everyone? Did you figure out why?

Reply:
Haven't figured it out yet :( But we should figure it out before final launch